essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -117,4 +117,14 @@ test = testGroup "Utility unit tests"
     , output1 = [Nothing, Just 2]
     , output2 = [Just 3, Just 4]
     }
+    , testProperty "traverse' works as expected for Lists"
+    $ \(inputs :: [[Int]]) -> fmap (fmap (+10)) inputs === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
+    , testProperty "traverse' works as expected for Maybes"
+    $ \(inputs :: [Maybe Int]) -> fmap (fmap (+10)) inputs === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
+    , testProperty "traverse' for [] is equal to resampleList"
+    $ \(inputs :: [[Int]]) -> fst (runIdentity $ steps (resampleList (arr (+10) :: Cell Identity Int Int)) inputs)
+      === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
+    , testProperty "traverse' for Maybe is equal to resampleMaybe"
+    $ \(inputs :: [Maybe Int]) -> fst (runIdentity $ steps (resampleMaybe (arr (+10) :: Cell Identity Int Int)) inputs)
+      === fst (runIdentity $ steps (traverse' (arr (+10) :: Cell Identity Int Int)) inputs)
   ]
```

Comment:
But these can be written with `CellSimulation`, right?

Reply:
indeed